concordances/cypher.go

```
@@ -91,7 +218,7 @@ func (pcw CypherDriver) ReadByAuthority(authority string, identifierValues []str
 	return processCypherQueryToConcordances(pcw, query, &results)
 }
 
-func processCypherQueryToConcordances(pcw CypherDriver, q *neoism.CypherQuery, results *[]neoResultStrunct) (concordances Concordances, found bool, err error) {
+func processCypherQueryToConcordances(pcw CypherDriver, q *neoism.CypherQuery, results *[]neoReadStruct) (concordances Concordances, found bool, err error) {
 	err = pcw.conn.CypherBatch([]*neoism.CypherQuery{q})
 	if err != nil {
 		log.Errorf("Error looking up Concordances with query %s from neoism: %+v\n", q.Statement, err)
```

should also return the actual error on the line below this one(225)